main.py

```diff
@@ -113,7 +113,8 @@ def make(self):
             for event in pygame.event.get():
                 # totaltime+=timer.tick()
                 if event.type == pygame.QUIT:
-                    crashed = True
+                    pygame.quit()
+                    sys.exit()
```

Comment:
In the sugargame test activity, QUIT is handled by returning.  Why do you handle it here by calling pygame.quit() and sys.exit()?  Doing so bypasses the normal Sugar activity exit path.